tools/proxy/proxy.js

```diff
@@ -1,3 +1,5 @@
+// @noflow
+/* eslint-disable */
```

Comment:
Is this needed now?

Reply:
Yes, both is needed. Apparently, eslint-disable doesn't act "early enough" for the flow-lint rule. Also, the file contains other eslint issues.